packages/gatsby-transformer-screenshot/README.md

```diff
@@ -83,3 +83,19 @@ First, you will need to [create a S3 bucket](https://docs.aws.amazon.com/AmazonS
 To build the Lambda package, run `npm run build-lambda-package` in this directory. A file called `lambda-package.zip` will be generated - upload this as the source of your AWS Lambda. Finally, you will need to set `S3_BUCKET` as an environment variable for the lambda.
 
 To set up the HTTP interface, you will need to use AWS API Gateway. Create a new API, create a new resource under `/`, select "Configure as proxy resource", and leave all the settings with their defaults. Create a method on the new resource, selecting "Lambda Function Proxy" as the integration type, and fill in the details of your lambda.
+
+## Placeholder image
+
+If your site pulls a lot of screenshots it might be beneficial to use placeholder image instead of downloading and processing all the screenshots. It will help with data sourcing and query running times.
+
+You can use placeholder image by setting `GATSBY_SCREENSHOT_PLACEHOLDER` environment variable when running `gatsby develop`:
+
+```shell
+GATSBY_SCREENSHOT_PLACEHOLDER=1 gatsby develop
```

Comment:
Can we use `GATSBY_SCREENSHOT_PLACEHOLDER=true`? I'm not a huge shell scripter, but that seems to be clearer?

Reply:
yeah, I'll switch to that - seems cleaner indeed. It just need to be truthy